src/controllers/verifyCommand.ts

```diff
@@ -13,9 +13,7 @@ export async function verifyCommand(
   env: env
 ) {
   const token = await generateUniqueToken();
-
-  const response = await sendUserDiscordData(
-    token,
+  const response = await sendUserDiscordData(    token,
```

Comment:
is there too much whitespace before `token`

Reply:
fixed automatically, via eslint